### ISSUE TITLE
Can't Enter World

### ISSUE THREAD

**Issue by Sasquashxx** [GrowtopiaNoobs/GrowtopiaServer#1297]
![unknown](https://user-images.githubusercontent.com/63276205/78830996-e3bc5200-79b6-11ea-96d0-2ae7d67632b5.png)

Anyone know why this shit happens

I appreciate the help.

**Comment by cmdocmd** [comment]
dont use leaked cpp use gt noobs source code it doesnt have this bug....

**Comment by Kazuto12x** [comment]
It's legit my cpp it's not leaked lmao

**Comment by LaYellow** [comment]
you should know the issue if youre the source owner.If its leaked,try to rewrite your own by GrowtopiaNoobs source

**Comment by Kazuto12x** [comment]
Im sure I messed something up which is why I'm asking if anyone has any ideas.

**Comment by Kazuto12x** [comment]
This wasn't a problem before I'm not sure what happened

**Comment by cmdocmd** [comment]
yes you missed with the source code but we dont know what did you do ? can you please show us what did you do before this happen ?

**Comment by GrowtopiaNoobs** [comment]
Revert your source code to original one or get app which can show you difference in files and post difference between current and original source here and we will be maybe able to help you.
